test/command/pod.spec.ts

```diff
@@ -0,0 +1,113 @@
+import { describeCommand, invokeTestCli } from '../utility'
+import { createFdp, getRandomString } from '../utils'
+
+describeCommand(
+  'Test Pod command',
+  ({ consoleMessages }) => {
+    it('should list pods', async () => {
+      const account = getRandomString()
+      const accountPassword = getRandomString()
+      const podName1 = getRandomString()
+      const podName2 = getRandomString()
+      const fdp = createFdp()
+
+      const wallet = fdp.account.createWallet()
+      await fdp.personalStorage.create(podName1)
+      await fdp.personalStorage.create(podName2)
+
+      await invokeTestCli([
+        'account',
+        'import',
+        wallet.mnemonic.phrase,
+        '--name',
+        account,
+        '--password',
+        accountPassword,
+      ])
+      consoleMessages.length = 0
+
+      await invokeTestCli(['pod', 'list', '--account', account, '--password', accountPassword])
```

Comment:
let's do a main account feature in another PR, where there is an additional property in the config that saves the main account's name. every time when the `--account` flag is not used, it can be the default value.
This property should be automatically set at the first created/imported account, but we need a feature that can modify this initial value.
On delete account command operation, we need to check whether the user wants to remove their main account or not